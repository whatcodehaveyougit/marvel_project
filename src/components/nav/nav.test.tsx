import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import Nav from "./nav";
import { describe, it, expect } from "@jest/globals";

describe("Nav", () => {
  it("renders correctly with another character", () => {
    const tree = render(
      <BrowserRouter>
        <Nav />
      </BrowserRouter>
    ).asFragment();
    expect(tree).toMatchSnapshot();
  });
});
