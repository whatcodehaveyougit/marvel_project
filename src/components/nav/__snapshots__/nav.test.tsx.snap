// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav renders correctly with another character 1`] = `
<DocumentFragment>
  <div>
    <div>
      <h1
        class="text-center text-6xl"
      >
        <a
          class="text-blue-500"
          href="/"
        >
          Marvel Application!
        </a>
      </h1>
    </div>
    <div
      class="flex justify-center items-center space-x-6 pt-2 pb-2"
    >
      <div>
        <a
          class="text-black-500 text-2xl hover:underline"
          href="/"
        >
          Home
        </a>
      </div>
      <div>
        <a
          class="text-black-500 text-2xl hover:underline"
          href="/about"
        >
          About
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
